python/cudf/cudf/_fuzz_testing/utils.py

```diff
@@ -19,7 +45,11 @@ def _generate_rand_meta(obj, dtypes_list):
 
     for _ in range(num_cols):
         dtype = random.choice(dtypes_list)
-        null_frequency = random.uniform(0, 1)
+        null_frequency = (
+            random.uniform(0, 1)
```

Comment:
Given that these tests are run in a long loop is there any performance benefit to `np.random.uniform` vs just the standard `random` from python?

Later on we might be able to improve the performance of the tests as a whole by batching/vectorizing random data generation, or maybe even doing the data generation asynchronously. Definitely out of scope for this PR though.

Reply:
Looks like using numpy for this specific api takes more time:


```python
In [1]: import numpy as np                                                                                                                                                                                                        

In [2]: import random                                                                                                                                                                                                             

In [3]: %timeit random.uniform(0, 1)                                                                                                                                                                                              
203 ns ± 2.11 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit np.random.uniform(0, 1)                                                                                                                                                                                           
1.79 µs ± 36.9 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
👍